webcurator-webapp/build.gradle

```diff
@@ -55,7 +55,7 @@ dependencies {
 
     implementation 'com.h2database:h2:1.4.199'
     implementation 'mysql:mysql-connector-java:5.1.6'
-    implementation 'com.oracle:ojdbc6:11.2.0.3'
+    implementation 'com.oracle:ojdbc6:12.1.0.2'
```

Comment:
My gradle complains about this version:

"Could not find com.oracle:ojdbc6:12.1.0.2"

Maybe that dependency was already in your local m2 repo? This version doesn't seem to be in the standard repos: https://mvnrepository.com/artifact/com.oracle/ojdbc6/12.1.0.2, it's in a repo at icm.edu.

Reply:
Ok I'll revert it back to 11.2.0.3 and we'll change the driver manually

Reply:
Alternatively, we could include a recent version of ojdbc8, since those are all backward compatible up to Oracle 11.2? These artefacts are available in central: https://mvnrepository.com/artifact/com.oracle.database.jdbc/ojdbc8. Version 18.3 of ojdbc8 supports JDK 8, 9, 10 and 11. See: https://www.oracle.com/database/technologies/faq-jdbc.html. Apparently, version 19 doesn't support JDK 10.

OTOH, if including the old 11.2 driver doesn't break Oracle support I'm okay with leaving it as it is.